tensorflow_probability/examples/bayesian_neural_network.py

```diff
@@ -74,15 +81,15 @@
                      default=50,
                      help="Network draws to compute predictive probabilities.")
 flags.DEFINE_bool("fake_data",
-                  default=None,
+                  default=False,
                   help="If true, uses fake data. Defaults to real data.")
 
 FLAGS = flags.FLAGS
 
 
 def plot_weight_posteriors(names, qm_vals, qs_vals, fname):
   """Save a PNG plot with histograms of weight means and stddevs.
-
+  
```

Comment:
remove whitespace

Reply:
I think you previous comment asked me to keep this line. Should there be space between the comment and args or no?

Reply:
Yeah you should be keep the line but not introduce any trailing whitespaces.